lib/main.js

```diff
@@ -31,14 +31,17 @@ function log (message /*: string */) {
 const NEWLINE = '\n'
 const RE_INI_KEY_VAL = /^\s*([\w.-]+)\s*=\s*(.*)?\s*$/
 const RE_NEWLINES = /\\n/g
+const NEWLINES_MATCH = /\n|\r|\r\n/
 
 // Parses src into an Object
 function parse (src /*: string | Buffer */, options /*: ?DotenvParseOptions */) /*: DotenvParseOutput */ {
   const debug = Boolean(options && options.debug)
   const obj = {}
 
-  // convert Buffers before splitting into lines and processing
-  src.toString().split(NEWLINE).forEach(function (line, idx) {
```

Comment:
`split` accepts a regular expression. did you try setting `NEWLINE` to your new `/\n|\r|\r\n/`? I'm hoping that's the only change needed 😺

Reply:
Oh, havent thought about that split accepts a regular expression. That would make it simpler. Adding it!

Reply:
One thought i have is, if the src variable is used outside of split (it isn't currently) it could cause issues when not replacing all the EOLs with a standard one. Such as the `NEWLINE` variable.

But as it is currently it works fine.